packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/edit-connector/edit-connector.controller.js

```diff
@@ -0,0 +1,95 @@
+export default class EditConnectorCtrl {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage, DatabaseService) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.DatabaseService = DatabaseService;
+    this.getErrorsMessages = EditConnectorCtrl.getErrorsMessages;
+  }
+
+  $onInit() {
+    this.messageContainer =
+      'pci.projects.project.storages.databases.dashboard.connectors.edit';
+    this.loadMessages();
+    this.trackDashboard('connectors::edit', 'page');
+    this.model = {
+      ...this.connector.configuration,
+    };
+  }
+
+  loadMessages() {
+    this.CucCloudMessage.unSubscribe(this.messageContainer);
+    this.messageHandler = this.CucCloudMessage.subscribe(
+      this.messageContainer,
+      { onMessage: () => this.refreshMessages() },
+    );
+  }
+
+  refreshMessages() {
+    this.messages = this.messageHandler.getMessages();
+  }
+
+  getModelValue() {
+    const configuration = {};
+    Object.keys(this.model).forEach((field) => {
+      if (![null, undefined, ''].includes(this.model[field])) {
```

Comment:
```suggestion
      if (this.model[field]) {
```

Reply:
This would not work, as we want to keep some `falsy` values like : 
-  `this.model[field] = 0`
-  `this.model[field] = false`